halo2_proofs/src/poly/domain.rs

```diff
@@ -245,7 +264,13 @@ impl<F: WithSmallOrderMulGroup<3>> EvaluationDomain<F> {
 
         self.distribute_powers_zeta(&mut a.values, true);
         a.values.resize(self.extended_len(), F::ZERO);
-        best_fft(&mut a.values, self.extended_omega, self.extended_k);
+        best_fft(
+            &mut a.values,
+            self.extended_omega,
+            self.extended_k,
+            &FFTData::default(),
```

Comment:
```suggestion
            &self.extended_fft_data,
```

Don't think this would otherwise work correctly with the recursive FFT?

Reply:
My mistake, thanks.  Fixed in latest.